js/components/portal-dashboard/all-responses-popup/popup-header.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+import css from "../../../../css/portal-dashboard/all-responses-popup/popup-header.less";
+interface IProps {
+    handleCloseAllResponsesPopup: (show: boolean) => void;
+}
+interface IState {
+    inFeedbackMode: boolean;
+  }
+
+export class PopupHeader extends React.PureComponent<IProps,IState>{
+
+    state = {
+            inFeedbackMode: false
+    };
+
+    render() {
+        // const activityName = this.props.activityName;  Activity name should be passed in
+        const activityName = "Hurricane Module V2 Activity 1: Hurricane Risk";
+        let feedbackToggleClass: any,
+            popupHeaderClass: any,
+            responsesToggleClass: any,
+            closeIconClass: any;
+        if (!this.state.inFeedbackMode) {
+                feedbackToggleClass =css.feedbackToggle+ " " + css.toggleOff;
```

Comment:
Ah, I think both of these new tsx modules should be using 2 spaces for a tab to be in line with the rest of the project.